### ISSUE TITLE
Docs Don't Match Site

### ISSUE THREAD

**Issue by donnfelker** [hotwired/stimulus#496]
The docs on the stimulus.hotwired.dev dev handbook are out of date in the `Designing for Resilience` section. 

In the "[Designing for Resilience](https://stimulus.hotwired.dev/handbook/designing-for-resilience)" section the docs show this: 

> Start by adding class="clipboard-button" to the button element:
`<button data-action="clipboard#copy" class="clipboard-button" data-clipboard-supported-class="clipboard--supported">Copy to Clipboard</button>`

**Important: Note that the data-clipboard-supported-class is on the button element, this is incorrect.**

This does not work (as I found through trial and error).

The docs here: https://github.com/hotwired/stimulus/blob/main/docs/handbook/04_designing_for_resilience.md
Show the following (which is correct): 

> We start off by adding data-clipboard-supported-class="clipboard--supported" to the div element that has the data-controller attribute:
```<div data-controller="clipboard" data-clipboard-supported-class="clipboard--supported">```

> Then add class="clipboard-button" to the button element:
```<button data-action="clipboard#copy" class="clipboard-button">Copy to Clipboard</button>```

The docs on the live site show adding the `data-clipboard-supported-class` to the button, but it should be added to the parent div. 

If added to the button, the user will get the following in the dev console: `Missing attribute "data-clipboard-supported-class"`

TLDR: Site needs to be updated with updated docs. :)

**Comment by dhh** [comment]
Fixed.
